main.cpp

```diff
@@ -56,10 +57,59 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    //int variables
+    int negativeNumber = -101;
+    int distanceMoved = 10;
+    int averageSpeed = 40;
+    ignoreUnused(negativeNumber);
+    ignoreUnused(distanceMoved);
+    ignoreUnused(averageSpeed);
+    
+    
+    
+    
+    
+    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+
+    //float variables
+    float outputGain = 1.0f;
+    float inputVolume = 0.5f;
+    float amplitude = -0.01f;
+    ignoreUnused(outputGain);
+    ignoreUnused(inputVolume);
+    ignoreUnused(amplitude);
+
+    //double variables
+    double firstDouble = 5.03;
+    double nextDouble = -0.2901;
+    double lastDouble = 9.991;
+    ignoreUnused(firstDouble);
+    ignoreUnused(nextDouble);
+    ignoreUnused(lastDouble);
+
+    //char variables
+    char firstLetter = 'a';
+    char lastLetter = 'z';
+    char secondLetter = 'b';
+    ignoreUnused(firstLetter);
+    ignoreUnused(lastLetter);
+    ignoreUnused(secondLetter);
+    
+    //bool variables
+    bool isPresent = true;
+    bool refresh = false;
+    bool reset = true;
+    ignoreUnused(isPresent);
+    ignoreUnused(refresh);
+    ignoreUnused(reset);
+
+    //unsigned int
+    unsigned int maxVal = 65535;
+    unsigned int minVal = 0;
+    unsigned int randomVal = 10002;
+    ignoreUnused(maxVal);
+    ignoreUnused(minVal);
+    ignoreUnused(randomVal);

```

Comment:
the instructions said:
`b) at the end of the function, pass each variable to a single call of the ignoreUnused function. see line 71 for an example`
- [ ] revise